server/src/api/FriendRoutes.ts

```diff
@@ -84,12 +97,20 @@ router.post("/", authenticateToken, async (req, res) => {
 
 router.put("/approve-request", authenticateToken, async (req, res) => {
   const userId = req.body.user.id;
-  const friendId = req.body.destination_friend_id;
+  const friendEmail = req.body.email;
```

Comment:
it would probably be better to use user id in this case, because when this endpoint is run we'll have access to the user id at this point.

Reply:
Fair I was thinking the same